scripts/test-nanvixd.sh

```diff
@@ -70,32 +46,42 @@ if ! ss -tln | grep -q ":${NANVIXD_PORT_NUMBER} "; then
 fi
 
 # Run a client.
-CURL_STDOUT_FILE_NAME="${LOGS_DIR}/curl-stdout_$(date "+%Y_%m_%d_%H_%M").log"
-curl \
+TENANT_ID="foo"
+APP_NAME="bar"
```

Comment:
Could you move these constants to the begin of the file so that they are more visible?